pyproject.toml

```diff
@@ -3,9 +3,7 @@ name = "subhkl"
 description = "Solving crystal orientation from Laue images"
 dynamic = ["version"]
 requires-python = ">=3.10"
-dependencies = [
-    "mantidworkbench >= 6.7.20230830",
```

Comment:
Quick check, this is not in PyPI that I am aware of [check search here](https://pypi.org/search/?q=mantidworkbench) an only available as a conda environment [see conda package here](https://anaconda.org/mantid/mantidworkbench).

So it makes sense to remove this from the pyproject toml.

Just sort of justifying removing the dependency here and instead using the conda and / or docker installed mantid workbench version :+1: